test-suite/CMakeLists.txt

```diff
@@ -236,16 +236,23 @@ if(DJINNI_WITH_PYTHON)
 
   # Prepare Python 3 environment
   find_package(PythonInterp "3" REQUIRED)
-  string(JOIN ":" PYTHONPATH
+  if (WIN32) # TODO, obviously that is not optimal
+    string(JOIN ";" PYTHONPATH
     "${CMAKE_SOURCE_DIR}/djinni/py"
     "${CMAKE_CURRENT_BINARY_DIR}/generated-src/python"
     $ENV{PYTHONPATH})
-
+  else()
+    string(JOIN ":" PYTHONPATH
+      "${CMAKE_SOURCE_DIR}/djinni/py"
+      "${CMAKE_CURRENT_BINARY_DIR}/generated-src/python"
+      $ENV{PYTHONPATH})
+  endif()
+  # TODO , we still do not find mylib, since it lands in a Debug subfolder of the current binary dir
```

Comment:
This TODO is WIN32 specific, right? I guess mylib is something from you local test setup? This todo would not be helpful to me if I didn't know the context. Can you improve the comment so it could be understood by a stranger? Maybe it makes sense to link the issue, to avoid having to explain the TODO in the code? 🤔

Reply:
This has meanwhile been fixed in my other developer branch ... I think I will merge those changes into this one soon, since they should also improve the Linux build (no more extraLIBRARY_PATH)

Reply:
could not fix the LIBRARY_PATH, but libs are now found on all platforms